keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -459,87 +463,123 @@ Windows privileged containers will be implemented with [Job Objects](https://doc
 
-Add functionality to Kuberuntime_sandbox to split out the linux sandbox creation and add windows sandbox creation (WIP PR). 
+Having a new field will also help us better anticipate / validate user intention.
 
-https://github.com/kubernetes/kubernetes/blob/a9f1d72e1de6450b890a0c0e451725468f54f515/pkg/kubelet/kuberuntime/kuberuntime_sandbox.go#L136 
+API validation will ensure that if `privileged` is set in `Container.SecurityContext.WindowsSercurityContextOptions` then `privileged` is also set in `PodSecurityContext.WindowsSecurityContextOptions`. Current behavior applies pod `WindowsSecurityContextOptions` to all container level `WindowsSecurityContextOptions` if not specified.
 
-#### <b>Host Network Mode</b>
+No additional validation will be performed against the existing `privileged` on `SecurityContext`.
 
-If using Windows privileged containers the Host Network Mode will also be enabled, as the pod will automatically get the host IP.  The PodSpec has a hostNetwork boolean field:
+#### Host Network Mode
 
-https://github.com/kubernetes/kubernetes/blob/d20fd4088476ec39c5ae2151b8fffaf0f4834418/pkg/kubelet/container/helpers.go#L227-L229
+Host Network mode for privleged Windows containers will always be enabled, as the pod will automatically get the host IP.
 
-https://github.com/kubernetes/kubernetes/blob/a9f1d72e1de6450b890a0c0e451725468f54f515/pkg/kubelet/kuberuntime/kuberuntime_sandbox.go#L98 
+Privileged Windows containers will be unable to align to pod namespaces due to limitiations in the Windows OS. This functionaility may likely be enabled in the future through a new KEP.
+
+Because of this we do not plan on performing **any** valiation around the existing `hostNetowrk` PodSpec field. This will give us the flexability to align privileged containers to host or pod namespaces in the future if/when that functionaility is added. Instead we will rely on documentation to explain 
+
+The PodSpec has a hostNetwork boolean field:
 
 In the case of windows this field would need to be addressed.  During the alpha stage the proposal for this field would be to add documentation that note the field is only applicable to Linux.  In addition the check for hostnetwork mode will need to be updated to check the privileged flag on windows PodSpec.  This will be implemented in the following way, with validation loosened in the future if required:
- - this pod must run on a windows host, and kubelets must reject it if not on windows hosts
- - all pods marked privileged on windows must have host network enabled, if not the pod does not validate
 
-In beta there is a possibility to enable the privileged container to be a part of a different network component.  If this feature is enabled we will use the existing Pod HostNetwork field to enable/disable.
+- this pod must run on a windows host, and kubelets must reject it if not on windows hosts
```

Comment:
> - this pod must run on a windows host, and kubelets must reject it if not on windows hosts

that seems inconsistent with multi-arch images, and is not something older kubelets will do.

Reply:
My updates here were to fix a markdown lint error. I remove this line as I find it a little confusing.